src/object/transform_cl.c

```diff
@@ -2158,6 +2178,30 @@ disk_to_domain2 (OR_BUF * buf)
       return NULL;
     }
 
+  if (vars[ORC_DOMAIN_SCHEMA_JSON_OFFSET].length > 0)
+    {
+      char *str = get_string (buf, vars[ORC_DOMAIN_SCHEMA_JSON_OFFSET].length);
```

Comment:
this looks fishy. first, I would avoid using get_string to get str. Maybe or_get_json_validator can be used or another way. Also avoid allocating yet another string for schema_raw.